File: sdks/java/src/main/java/com/decentrilicense/LicenseException.java
package com.decentrilicense;

/**
 * License Exception
 */
public class LicenseException extends Exception {
    public LicenseException() {
        super();
    }
    
    public LicenseException(String message) {
        super(message);
    }
    
    public LicenseException(String message, Throwable cause) {
        super(message, cause);
    }
    
    public LicenseException(Throwable cause) {
        super(cause);
    }
}

File: sdks/csharp/DecentriLicense/LicenseException.cs

using System;

namespace DecentriLicense
{
    public class LicenseException : Exception
    {
        public LicenseException(string message) : base(message)
        {
        }

        public LicenseException(string message, Exception innerException) : base(message, innerException)
        {
        }
    }
}

